auth0/resource_auth0_prompt_custom_text_test.go

```diff
@@ -19,49 +18,57 @@ func TestAccPromptCustomText(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "prompt", "login"),
 					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "language", "en"),
-					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "body", "{\n    \"login\": {\n        \"alertListTitle\": \"Alerts\",\n        \"buttonText\": \"Continue\",\n        \"emailPlaceholder\": \"Email address\"\n    }\n}"),
+					resource.TestCheckResourceAttr(
+						"auth0_prompt_custom_text.prompt_custom_text",
+						"body",
+						"{\n    \"login\": {\n        \"alertListTitle\": \"Alerts\",\n        \"buttonText\": \"Continue\",\n        \"emailPlaceholder\": \"Email address\"\n    }\n}",
+					),
 				),
 			},
 			{
 				Config: testAccPromptCustomTextUpdate,
 				Check: resource.ComposeTestCheckFunc(
 					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "prompt", "login"),
 					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "language", "en"),
-					resource.TestCheckResourceAttr("auth0_prompt_custom_text.prompt_custom_text", "body", "{\n    \"login\": {\n        \"alertListTitle\": \"Alerts\",\n        \"buttonText\": \"Continue\",\n        \"emailPlaceholder\": \"Email address\"\n    }\n}"),
+					resource.TestCheckResourceAttr(
+						"auth0_prompt_custom_text.prompt_custom_text",
+						"body",
+						"{\n    \"login\": {\n        \"alertListTitle\": \"Alerts\",\n        \"buttonText\": \"Proceed\",\n        \"emailPlaceholder\": \"Email Address\"\n    }\n}",
+					),
 				),
 			},
 		},
 	})
 }
 
 const testAccPromptCustomTextCreate = `
-
 resource "auth0_prompt_custom_text" "prompt_custom_text" {
   prompt = "login"
   language = "en"
-  body = <<EOF
-{
-"login": {
-	"alertListTitle": "Alerts",
-	"buttonText": "Continue",
-	"emailPlaceholder": "Email address"}
-}
-EOF
+  body = jsonencode(
```

Comment:
I changed this as well to use jsonencode so we can write and format the json payload as an object in HCL.

Reply:
nice. didn't know about that